### PR TITLE
Druva c2c

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+**To configure a Druva Source**
+
+1. In Sumo Logic, select **Manage Data** > **Collection** > **Collection**.
+1. On the **Collectors** page, click **Add Source** next to a Hosted Collector.
+1. Select **Druva**. <br/><img src={useBaseUrl('img/send-data/druva-icon.png')} alt="druva-icon.png" width="200"/>
+1. **Name**. Enter a Name to display for the Source in the Sumo Logic web application.
+1. **Description**. (Optional)
+1. **Source Category**.(Optional) Enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
+1. 1. Fields. (Optional) Click **+Add** to ad additional fields; each field needs a name (key) and value.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md:83]:
```suggestion
1. Fields. (Optional) Click **+Add** to add additional fields; each field needs a name (key) and value.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists and is enabled in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist, or is disabled, in the Fields table schema. In this case, an option to automatically add or enable the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo that does not exist in the Fields schema or is disabled it is ignored, known as dropped.
+8. **Base URL**. Choose the base url from the drop-down menu as per your requirements — [Data Governance Cloud](https://apis.druva.com) and [Data GovernanceGov Cloud](https://govcloudapis.druva.com).<br/><img src={useBaseUrl('img/send-data/base-url.png')} alt="base-url.png" width="400"/>
+9. **Client ID**. Enter your Client ID generated from your [Druva application](https://developer.druva.com/docs/authentication).
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md:87]:
```suggestion
1. **Base URL**. Choose the base url from the drop-down menu as per your requirements — [Data Governance Cloud](https://apis.druva.com) and [Data GovernanceGov Cloud](https://govcloudapis.druva.com).<br/><img src={useBaseUrl('img/send-data/base-url.png')} alt="base-url.png" width="400"/>
1. **Client ID**. Enter your Client ID generated from your [Druva application](https://developer.druva.com/docs/authentication).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+### Config Parameters
+
+| Parameter | Type | Required | Description | Access |
+|:---|:---|:---|:---|:---|
+| `name` | String | Yes | Type the desired name of the Source and it must be unique per Collector. This value is assigned to the `metadata field _source`.  | modifiable |
+| `description` | String  | No | Type the description of the Source. | modifiable |
+| `category` | String | No | Type the category of the source. This value is assigned to the metadata field `_sourceCategory`. | modifiable |
+| `fields` | JSON Object | No | JSON map of key-value fields (metadata) to apply to the Collector or Source. Use the boolean field `_siemForward` to enable forwarding to SIEM. | modifiable |
+| `baseURL` | String | Yes | The base URL from which the customer wants to retrieve event data.. | modifiable |
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md:120]:
```suggestion
| `baseURL` | String | Yes | The base URL from which the customer wants to retrieve event data. | modifiable |
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+| `clientID` | String | Yes | Client ID key of an application. | modifiable |
+| `secretKey` | String | Secret key of an application | modifiable |
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md:122]:
```suggestion
| `secretKey` | String | Secret key of an application. | modifiable |
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist, or is disabled, in the Fields table schema. In this case, an option to automatically add or enable the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo that does not exist in the Fields schema or is disabled it is ignored, known as dropped.
+1. **Base URL**. Choose the base url from the drop-down menu as per your requirements — [Data Governance Cloud](https://apis.druva.com) and [Data GovernanceGov Cloud](https://govcloudapis.druva.com).<br/><img src={useBaseUrl('img/send-data/base-url.png')} alt="base-url.png" width="400"/>
+1. **Client ID**. Enter your Client ID generated from your [Druva application](https://developer.druva.com/docs/authentication).
+1. **Secret Key**. Enter your Client secret key generated within your [Druva Application](https://developer.druva.com/docs/authentication).<br/><img src={useBaseUrl('img/send-data/druva-config.png')} alt="druva-config.png" width="600"/>
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md:88]: Broken image. It looks like you accidentally put the images in the docs folder  (`/docs/send-data`). Please move them to `/static/img/send-data`.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+1. **Source Category**.(Optional) Enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
+1. Fields. (Optional) Click **+Add** to add additional fields; each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists and is enabled in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist, or is disabled, in the Fields table schema. In this case, an option to automatically add or enable the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo that does not exist in the Fields schema or is disabled it is ignored, known as dropped.
+1. **Base URL**. Choose the base url from the drop-down menu as per your requirements — [Data Governance Cloud](https://apis.druva.com) and [Data GovernanceGov Cloud](https://govcloudapis.druva.com).<br/><img src={useBaseUrl('img/send-data/base-url.png')} alt="base-url.png" width="400"/>
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md:86]: Broken image. It looks like you accidentally put the images in the docs folder  (`/docs/send-data`). Please move them to `/static/img/send-data`.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+* Obtain access tokens by [authentication](#authentication).
+
+
+### Configure Druva inSync to export events
+
+Only an Druva inSync Cloud administrator can enable the option to export events and define the Events API settings. To get the event logs, follow the steps to enable the **Export Events**:
+1. Log in to the Druva inSync Management Console with [inSync Cloud](https://login.druva.com/) account or [inSync GovCloud](https://loginfederal.druva.com/) account.
+2. On the [inSync Management Console](https://docs.druva.com/Endpoints/030_Set_up_inSync_for_Endpoints/010_Initial_Configuration/010_Sign_in_to_inSync_Management_Console/Sign_in_to_inSync_Management_Console) menu bar, click **Settings** (wheel icon) > Settings. The Settings page appears.
+3. Click the **inSync APIs** tab.
+4. In the Events API settings area, click **Edit**. The Edit Events API Settings window appears.<br/><img src={useBaseUrl('img/send-data/events-api-settings.png')} alt="events-api-settings.png" width="450"/>
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md:30]: Broken image. It looks like you accidentally put the images in the docs folder  (`/docs/send-data`). Please move them to `/static/img/send-data`.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+
+## Setup and configuration Source
+
+When you create a Druva Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector.md).
+
+**To configure a Druva Source**
+
+1. In Sumo Logic, select **Manage Data** > **Collection** > **Collection**.
+1. On the **Collectors** page, click **Add Source** next to a Hosted Collector.
+1. Select **Druva**. <br/><img src={useBaseUrl('img/send-data/druva-icon.png')} alt="druva-icon.png" width="200"/>
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md:79]: Broken image. It looks like image is in the docs folder  (`/docs/send-data`). Please move it to `/static/img/send-data`.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md (hunk 5 above)
Comment (JainM6) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md]: That's correct. I have fixed it. Thank you, Kim.

**Comment by keyur-sumo** [comment]
Yes, please close the old PR.

**Comment by kimsauce** [comment]
Closing in favor of new Druva C2C PR

**Pull request closed without merging**